Update ios_14.md (#1409)
Appboy/braze-docs: _docs/_developer_guide/platform_integration_guides/ios/ios_14.md

- Geofences are no longer supported by iOS for users who choose the new  _approximate location_ permission.
- Customers using the "Last Known Location" targeting features are required to upgrade their Braze iOS SDK to at least v3.26.1 for compatibility with _approximate location_ permission.
- IDFA collection now requires a permission prompt. Failure to update your IDFA collection code will result in a blank value as if a user declined to provide this permission.

> - Based on our testing, Apple no longer displays the provisional authorization dialog and instead sets users as "authorized". [Learn More](#push-authorization)

Author: ```suggestion
- Based on our testing, Apple may no longer displays the provisional authorization dialog and instead sets users as "authorized". [Learn More](#push-authorization)
```

> - Based on our testing, Apple may no longer display the provisional authorization dialog and instead set users as "authorized". [Learn More](#push-authorization)

Reviewer: ```suggestion
- Based on our testing, Apple may no longer display the provisional authorization dialog and instead set users as "authorized" as soon as apps request provisional authorization. [Learn More](#push-authorization)
```

> - Based on our testing, Apple may no longer display the provisional authorization dialog and instead set users as "authorized". [Learn More](#push-provisional-auth)

Author: ```suggestion
- Based on our testing, Apple may no longer display the provisional authorization dialog, and will instead set users as "authorized" as soon as the app requests provisional authorization. [Learn More](#push-authorization)
```

> - Based on our testing, Apple will no longer displaying the provisional authorization dialog and instead setting users' as "authorized". [Learn More](#push-authorization)

Author: ```suggestion
- Based on our testing, Apple no longer displays the provisional authorization dialog and instead sets users as "authorized". [Learn More](#push-authorization)
```

These IDFA updates take effect once end-users upgrade their device to iOS 14. Please ensure your app is using the new `AppTransparencyFramework` even if you do not explicitly target iOS 14 support.
{% endalert %}

> #### Changes to Braze IDFA collection 

Reviewer: ```suggestion
#### Changes to Braze IDFA collection
```
